source/manual/alerts/expiring-api-tokens-in-signon.html.md

```diff
@@ -60,7 +61,13 @@ for Publishing API expires in X days*. In this example, the API User is
 >
 > The api call could be something like
 >
-> `GdsApi::SupportApi.new(Plek.new.find("support-api"), bearer_token: ENV["SUPPORT_API_BEARER_TOKEN"])`
+> `gds_api = GdsApi::Router.new(Plek.new.find("router-api"), bearer_token: ENV["ROUTER_API_BEARER_TOKEN"])`
+>
+> `gds_api.get_route("/foo")`
+>
+> If the token works, then the above should get us something other than an unauthorised error.
+> You could test this out by setting up `gds_api` with an incorrect token, e.g. `bearer_token: "NOT_THE_REAL_TOKEN"`
+> and checking that this gives a different response to when it is set up with the new token.
```

Comment:
I feel like revoking the old token should be the final step after this one, and this step should be more directional, the words "once you're happy" seem too vague (i.e. check the new token works with this step --> revoke the old token). Also maybe worth putting in the bit about waiting for Puppet to perpetuate the token before revoking it?